dev/com.ibm.ws.jaxrs.2.1_fat_extended/test-applications/subResourceApp/src/jaxrs21/fat/subresource/SubResourceTestServlet.java

```diff
@@ -58,6 +58,42 @@ public void testSubResourceMsgBodyWriter(HttpServletRequest req, HttpServletResp
         assertEquals("{\"subId\":\"1\"}", result);
     }
 
+    @Test
+    public void testSubResourceContextInjection(HttpServletRequest req, HttpServletResponse resp) throws Exception {
+        Response response = target(req, "sub/context").request().get();
+        String result = response.readEntity(String.class);
+        assertEquals(200, response.getStatus());
+        System.out.println("result=" + result);
+        assertTrue(result.contains("/sub/context"));
+    }
+
+    @Test
+    public void testSubResourceCdiInjection(HttpServletRequest req, HttpServletResponse resp) throws Exception {
+        Response response = target(req, "subCdi/cdi").request().get();
+        String result = response.readEntity(String.class);
+        assertEquals(200, response.getStatus());
+        System.out.println("result=" + result);
+        assertTrue(result.contains("some string"));
+    }
+
+    @Test
+    public void testSubResourceContextInjectionWithGetResource(HttpServletRequest req, HttpServletResponse resp) throws Exception {
+        Response response = target(req, "subGet/context").request().get();
+        String result = response.readEntity(String.class);
+        assertEquals(200, response.getStatus());
+        System.out.println("result=" + result);
+        assertTrue(result.contains("/subGet/context"));
+    }
+
+//    @Test
+//    public void testSubResourceCdiInjectionWithGetResource(HttpServletRequest req, HttpServletResponse resp) throws Exception {
+//        Response response = target(req, "subGet/cdi").request().get();
+//        String result = response.readEntity(String.class);
+//        assertEquals(200, response.getStatus());
+//        System.out.println("result=" + result);
+//        assertTrue(result.contains("some string"));
+//    }
```

Comment:
Why comment this out vs delete?

Reply:
I'll add an additional comment to the code, but the intent was to show that this scenario is not intended to work.